backend/app/models/assistants.py

```
@@ -1,11 +1,14 @@
 from datetime import datetime
-from typing import Optional, List
-from sqlmodel import Field, Relationship, SQLModel
-from sqlalchemy import Column, String
+from typing import List, Optional
```

Ruff (0.12.2)

`typing.List` is deprecated, use `list` instead

(UP035)